Polygon_mesh_processing/doc/Polygon_mesh_processing/Concepts/PMPAutorefinementVisitor.h

```diff
@@ -0,0 +1,25 @@
+/// \ingroup PkgPolygonMeshProcessingConcepts
+/// \cgalConcept
+///
+/// The concept `PMPAutorefinementVisitor` defines the requirements for the visitor
```

Comment:
This concept is not mentioned on the small feature page.
It is also not in the PackageDescription.txt